seatunnel-e2e/seatunnel-connector-v2-e2e/connector-hbase-e2e/pom.xml

```
@@ -26,14 +26,23 @@
     <name>SeaTunnel : E2E : Connector V2 : Hbase</name>
 
     <dependencies>
-
+        <dependency>
+            <groupId>org.apache.seatunnel</groupId>
+            <artifactId>connector-fake</artifactId>
+            <version>${project.version}</version>
```

```suggestion
            <version>${project.version}</version>
            <scope>test</scope>
```

Do you need to maintain the same style?

---

Thanks for your advice. I have already modified it.